packages/server/package-lock.json

```diff
@@ -1,30 +0,0 @@
-{
-  "name": "@koh/server",
-  "version": "0.0.1",
-  "lockfileVersion": 1,
-  "requires": true,
```

Comment:
Was this file meant to be deleted?

Reply:
yes, package-lock.json is like yarn.lock, auto generated. We shouldn't track that